protocols/ntree/simulation.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 func init() {
+	sda.RegisterNewProtocol("CoSiNtree", func(node *sda.Node) (sda.ProtocolInstance, error) { return NewProtocol(node) })
```

Comment:
Sure. I simply changed from `ProtocolRegisterName` to `RegisterNewProtocol`, so I guess I must have taken that from a  old version :D